tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/impl/MigrationTo200.java

```diff
@@ -246,6 +253,109 @@ private Set<String> getSetOfScopes(Set<String> indices) throws IOException {
         return scopes;
     }
 
+    private void createProfileAliasDocumentsFromProfile() throws IOException {
+        System.out.println("Migration \"Create profileAlias from profile\" started");
+        Instant migrationTime = Instant.now();
+        int initialOffset = 1000;
+        int size = 1000;
+        doProcessResponse(migrationTime, initialOffset, size, null);
+        System.out.println("Migration \"Create profileAlias from profile\" completed.");
+    }
+
+    private void doProcessResponse(Instant migrationTime, int initialOffset, int offset, String scrollId) throws IOException {
```

Comment:
I think we should rename this method to something like processProfiles or something like that, because we are not processing a response here.